lib/examples/axum/src/main.rs

```diff
@@ -20,17 +24,25 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
 	db.use_ns("namespace").use_db("database").await?;
 
-	let router = Router::new()
-		.route("/person/:id", post(person::create))
-		.route("/person/:id", get(person::read))
-		.route("/person/:id", put(person::update))
-		.route("/person/:id", delete(person::delete))
-		.route("/people", get(person::list))
-		.with_state(db);
-
 	let listener = TcpListener::bind("localhost:8080").await?;
+	let router = create_router(db);
 
 	axum::serve(listener, router).await?;
 
 	Ok(())
 }
+
+fn create_router(db: Surreal<surrealdb::engine::any::Any>) -> Router {
```

Comment:
Having `create_router` in `main.rs` is fine too but that will require integration tests to be inside `/src`. Integration tests should typically be in `/tests`, as per Rust conventions. Consider moving this to `lib.rs`.